### DISCUSSION TITLE
get api key for extension

### DISCUSSION THREAD

**Discussion by notrbm** [Authenticator-Extension/Authenticator#1025] [General]
### Describe the issue

How TO Get Api key for extension

### Browser

Chrome

### Browser Version

Latest

### Extension Version

Latest

**Comment by mymindstorm** [comment]
Hi,

We aren’t able to provide our API keys due to agreements with the API providers. You should be able to sign up as a developer and generate your own keys.

On Feb 22, 2023, at 2:20 PM, MichaelRb ***@***.***> wrote:

﻿

Describe the issue

How TO Get Api key for extension

Browser

Chrome

Browser Version

Latest

Extension Version

Latest

**Comment by notrbm** [comment]
How Where Which Website Do I Get It Please Explain
